language/go/embed.go

```diff
@@ -156,24 +156,30 @@ func (er *embedResolver) resolve(embed fileEmbed) (list []string, err error) {
 		}
 	}()
 
+	glob := embed.path
+	all := strings.HasPrefix(embed.path, "all:")
+	if all {
+		glob = embed.path[len("all:"):]
```

Comment:
I think this would be better as `strings.TrimPrefix`

Reply:
done